opal/util/cmd_line.c

```diff
@@ -499,7 +499,7 @@ char *opal_cmd_line_get_usage_msg(opal_cmd_line_t *cmd)
     int argc;
     size_t j;
     char **argv;
-    char *ret, temp[MAX_WIDTH * 2], line[MAX_WIDTH * 2];
+    char *ret, temp[(MAX_WIDTH * 2) - 1], line[MAX_WIDTH * 2];
```

Comment:
I think this is wrong.  Please see line 577.  I think it should use `strlen` there instead of `sizeof`

Reply:
Can you elaborate?

`temp` get's strcatted into `line`. To leave room for a null terminator, it should be one less that `line`.

I guess alternatively we could increase `line`'s size by one.

Reply:
We discussed offline the issues with multiple strncats into the same buffer, and austen's reworking.